### DISCUSSION TITLE
Hover over Nickname

### DISCUSSION THREAD

**Discussion by michallanda** [EssentialsX/Essentials#4317] [Ideas and Feature Suggestions] [answered]
Would it be possible to add a hover over display whenever someone has a nick? So if someone has a nickname and you hover over their name in chat, it'll show who that actually is? It would make it a lot easier to know who is saying what if someone changes their nickname to something not similar to their IGN, without having to do /realname (user).

**Comment by OliveOilzz** [comment]
This would be a great feature for everyone but especially server staff so they can easily find someone's real name. Please add this

**Comment by molor** [comment, marked as answer]
Probably the same reply as in https://github.com/EssentialsX/Essentials/discussions/4054#discussioncomment-683437

**Comment by pop4959** [comment]
This is also being tracked in https://github.com/EssentialsX/Essentials/issues/2391.
